lib/Stats.js

```diff
@@ -576,6 +593,16 @@ class Stats {
 			});
 			table(t, "rrrlll");
```

Comment:
Could you hide the table when it's empty? I guess the headers are always displayed. This would look weird when no assets are displayed.